draft-rosomakho-httpbis-h3-unbound-data.md

```diff
@@ -95,7 +96,7 @@ UNBOUND_DATA Frame {
 
 The `UNBOUND_DATA` frame has no payload. The Length field of the frame MUST be zero. If a nonzero length is received, the endpoint MUST treat this as a connection error of type `H3_FRAME_ERROR`.
 
-The `UNBOUND_DATA` frame is only valid on request or response streams. It is invalid on control streams, QPACK encoder/decoder streams, or push streams. If an endpoint receives an `UNBOUND_DATA` frame on a stream that isn't a client-initiated bidirectional stream, it MUST treat it as a connection error of type `H3_FRAME_UNEXPECTED`.
+The `UNBOUND_DATA` frame is only valid on request or response streams. It is invalid on unidirectional streams. If an endpoint receives an `UNBOUND_DATA` frame on a stream that isn't a client-initiated bidirectional stream, it MUST treat it as a connection error of type `H3_FRAME_UNEXPECTED`.
```

Comment:
UNBOUND_DATA is also invalid on server-initiated bidirectional streams. I would just remove the sentence

```suggestion
The `UNBOUND_DATA` frame is only valid on request or response streams. If an endpoint receives an `UNBOUND_DATA` frame on a stream that isn't a client-initiated bidirectional stream, it MUST treat it as a connection error of type `H3_FRAME_UNEXPECTED`.
```